src/components/Timeline/Event/Events.js

```diff
@@ -0,0 +1,77 @@
+import React from 'react'
+import { connect } from 'react-redux'
+import PropTypes from 'prop-types'
+
+import ErrorMessage from 'components/elements/ErrorMessage'
+import LoadingMessage from 'components/elements/LoadingMessage'
+import Highlight from 'components/elements/Highlight'
+import EventCard from 'components/Timeline/Event/EventCard'
+import BootstrapPlaybook from 'components/Timeline/Playbook/BootstrapPlaybook'
+import * as eventTypeActions from 'actions/eventTypeActions'
+
+export const Events = ({events, ticketId, incidentId}) => <div>
+  {Array.from(events)
```

Comment:
Are we expecting `events` to have a type other than Array? 

Would the code below give the same results?
``` javascript
  {
    const connectEvent = (event) => <ConnectedEvent 
      key={event.id} 
      event={event} 
      ticketId={ticketId} 
    />

    Array.from(events).map(event => connectEvent(event))
  }
```

Reply:
Events should always be an array.

That code would give the same result. Do you think that improves readability? Are there other benefits from that format? I'm somewhat partial to inlining the components in map statements within JSX.